Samples/Scratchpad/scratchpad_c/generated/ScratchpadCSharp.h

```diff
@@ -13,50 +13,50 @@
 #include "fn_pointer_types.h"
 #include "ScratchpadCSharp_Dependency.h"
 
-typedef struct {
+typedef struct ScratchpadCSharp_SimpleFunctions_Vector3 {
```

Comment:
Need to investigate if the type name duplicated here is actually needed or not.

Reply:
I'm confused, I thought you could do `void Foo (MyStruct* instance)` with just `typedef struct` and not being named afer that.  Most of the code doesn't seem to need it so far.

Reply:
Nevermind I see now.  The latter allows you to do both `MyStruct` and `struct MyStruct` where the former only allows the former only.